src/compiler/comments.ts

```diff
@@ -274,7 +274,7 @@ namespace ts {
             if (hasTrailingNewLine) {
                 writer.writeLine();
             }
-            else {
+            else if (_kind === SyntaxKind.MultiLineCommentTrivia) {

```

Comment:
Remove the leading underscore (`_`) prefix from `_kind` here and in the parameter.